server/consts/index.ts

```diff
@@ -1,4 +1,4 @@
 import ERRORS from "./errors";
 import * as MESSAGES from "./message";
-
 export { MESSAGES, ERRORS };
+export * as REASON_CODE from "./reasonCode";
```

Comment:
This one should be in the line 3 and need to be exported as the part of above object.